photon-api/src/main/scala/com/linkedin/photon/ml/estimators/GameEstimator.scala

```diff
@@ -248,6 +308,13 @@ class GameEstimator(val sc: SparkContext, implicit val logger: Logger) extends P
     // Verify valid GameEstimator settings
     validateParams()
 
+    // Locked coordinates should not be present in the optimization configurations
```

Comment:
I think it makes sense that the locked coordinates shouldn't have coordinate configurations explicitly specified for the following reasons:
- We don't need this information - we're enforcing that user configuration files should not have unnecessary and potentially confusing information
- If we need it, we can get it from the metadata stored with each model
- For the above reason, we would want to avoid mismatches between historic and new configuration data

I'm okay with modifying this if need be later, though for now I think we should do it this way. I think it's better to be too strict and become more lenient, than to be too lenient from the start.

Reply:
Thanks for the explanation. Should be fine to leave it for now.